File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        set<int>s(nums.begin(), nums.end());
        int n = nums.size(), size = 0, maxsize = 0;
        for(int i = 0; i < n; i++) {
            int prev = nums[i];
            size = 1;
            while(s.find(prev+1) != s.end()){
                prev = prev+1;
                size++; 
            }
            maxsize = max(size, maxsize);
        }
        return maxsize;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        // Hashmap put 
        //sort it by frequency -> find top k
        HashMap<Integer, Integer> mp = new HashMap<>();
        for(int x : nums) mp.put(x, mp.getOrDefault(x, 0)+1);
        List<Integer> sortedList = new ArrayList<>(mp.keySet());
        sortedList.sort((a,b)->mp.get(b)-mp.get(a));
        int ans[] = new int[k];
        for(int i = 0; i < k; i++) {
            ans[i] = sortedList.get(i);
        }
        return ans;
    }
}
